tox.ini

```diff
@@ -597,30 +597,76 @@ commands =
 [testenv:docker-tests]
 basepython: python3
 deps =
-  pip >= 20.3.3
-  pytest
+  aiopg==1.2.1
```

Comment:
```suggestion
  aiopg==1.4.0
```

Reply:
It should be fine, instrumentation supports aiopg < 2

Reply:
Fixed